src/commands/FxTweet.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  CommandInteraction,
+  Client,
+  ApplicationCommandType,
+  ApplicationCommandOptionType,
+  InteractionResponse,
+} from 'discord.js';
+
+import { Command } from '../interfaces/Command';
+
+export const FxTweet: Command = {
+  name: 'fxtweet',
+  description: 'Fix a tweet video embed',
+  type: ApplicationCommandType.ChatInput,
+  options: [
+    {
+      name: 'tweet',
+      description: 'Twitter URL',
+      type: ApplicationCommandOptionType.String,
+      required: true,
+    },
+  ],
+  run: (
+    client: Client,
+    interaction: CommandInteraction,
+  ): Promise<InteractionResponse<boolean>> => {
```

Comment:
Nit: `Command` already infers callback's types!
```suggestion
import { ApplicationCommandType, ApplicationCommandOptionType } from 'discord.js';

import { Command } from '../interfaces/Command';

export const FxTweet: Command = {
  name: 'fxtweet',
  description: 'Fix a tweet video embed',
  type: ApplicationCommandType.ChatInput,
  options: [
    {
      name: 'tweet',
      description: 'Twitter URL',
      type: ApplicationCommandOptionType.String,
      required: true,
    },
  ],
  run: (_, interaction) => {
```